src/test/java/com/cloudant/tests/ResponseTest.java

```diff
@@ -0,0 +1,86 @@
+package com.cloudant.tests;
+
+import static org.junit.Assert.assertEquals;
+
+import com.cloudant.client.api.model.Response;
+import com.cloudant.client.org.lightcouch.CouchDbException;
+import com.cloudant.client.org.lightcouch.DocumentConflictException;
+import com.cloudant.client.org.lightcouch.NoDocumentException;
+import com.cloudant.tests.util.CloudantClientResource;
+import com.cloudant.tests.util.DatabaseResource;
+import com.cloudant.tests.util.TestLog;
+
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Test cases to verify status code from Response object.
+ * Assert status codes in CouchDbException and its subclasses.
+ */
+public class ResponseTest {
+    @ClassRule
+    public static final TestLog TEST_LOG = new TestLog();
```

Comment:
Doesn't look like there is any logging in this test so probably don't need this.